infrastructure/laa-estimate-financial-eligibility-for-legal-aid-uat/ingress-dashboard.yaml

```diff
@@ -349,6 +349,207 @@ data:
             "y": 30
           },
           "hiddenSeries": false,
+          "id": 7,
+          "legend": {
+            "alignAsTable": false,
+            "avg": false,
+            "current": true,
+            "max": false,
+            "min": false,
+            "rightSide": true,
+            "show": true,
+            "total": false,
+            "values": true
+          },
+          "lines": true,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null",
+          "options": {
+            "alertThreshold": true
+          },
+          "percentage": false,
+          "pluginVersion": "9.2.4",
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": true,
+          "steppedLine": false,
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "prometheus"
+              },
+              "editorMode": "code",
+              "expr": "sum by(status) (rate(nginx_ingress_controller_requests{exported_namespace=\"$namespace\", path!~\"/socket.io/.*\", ingress=\"$ingress\", status=~\"4..|5..\"}[5m]))",
```

Comment:
Do you want to try this: https://dsdmoj.atlassian.net/wiki/spaces/LE/pages/4258463754/Grafana+admin to try and make sense of the y axis?

Reply:
@davidread good idea! I've tried that change via the Grafana editor, and it looks to me to be working as intended, so have applied it here.